examples/default/init/main.tf

```diff
@@ -13,8 +13,7 @@ terraform {
 }
 
 provider "aws" {
-  version             = ">= 2.17"
-  region              = "eu-west-1"
+  region = "eu-west-1"
```

Comment:
This is incorrectly formatted with whitespace. `terraform fmt`

Reply:
this have been corrected